dagrunner/utils/visualisation.py

```diff
@@ -7,33 +7,66 @@
 """
 
 import base64
+import math
 import os
+import sys
+import warnings
+import webbrowser
+from typing import Iterable, Union
+
+import networkx as nx
+
+from . import as_iterable, in_notebook
+
+# set web component path to local file when running pytest
+if "pytest" in sys.modules:
+    WEBCOMPONENT_PATH = "../../../visual/mermaid-table-standard.js"
+else:
+    WEBCOMPONENT_PATH = "https://cdn.jsdelivr.net/gh/MetOffice/dagrunner@vvisual_tooltip_table/visual/mermaid-table-standard.js"
```

Comment:
Note that the web component path is relative when running pytest (this means the tests use the version from its own checkout).

Otherwise, the web component path is going via free CDN (jsdeliver), pointing to our repository.
Right now, this is pointing to this development branch.  After merge, I'll tag a release and update this path so that it points to this tag releasese.